omero/developers/Modules/Api.txt

```diff
@@ -7,6 +7,8 @@ connection e.g. Python:
 
 ::
 
+    import omero.all
```

Comment:
It's possible I've missed a discussion related to this on 5.3.x with @joshmoore and or @manics, maybe even we have to be more explicit about it, but I think you want `import omero.clients` here. That is the package that should be imported by clients.